src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -383,7 +383,7 @@ public function openAction(Request $request)
         $file = $request->query->get('file');
         $line = $request->query->get('line');
 
-        $filename = $this->baseDir.DIRECTORY_SEPARATOR.$file;
+        $filename = file_exists($file) ? $file : $this->baseDir.DIRECTORY_SEPARATOR.$file;
```

Comment:
We should be extremely careful with features like this to not introduce security vulnerabilities (somewhat related to #25296).

Reply:
I agree with you Javier
Yet it's already possible to access any file, using `../` in the "file" parameter, isn't it?

Reply:
From a security/profiler pov i think this shouldnt really hurt. Or at least is the quick win. Making it a relative path in FileLinkFormatter is not worth the hassle IMHO.

Reply:
@nicolas-grekas no, there's an explicit guard against using `/../` at any point in the path.

Reply:
oh, then I doubly agree with Javier, we should find another way @ro0NL

Reply:
Yeah.. i was reconsidering already. I can imagine we intentionally want to support files here relative to the project, _only_. Given that i see no reason to deny dotfiles (assuming it's sensitive or so..), making # #25296 a non-issue IMHO.